README.md

```diff
@@ -33,69 +47,225 @@
+    on phl_pwd_parcels
+    using GiST (ST_Transform(geometry, 32129));
+-- query & cross join
+SELECT 
+    p.address,
+    cl.stop_name,
+    cl.distance_m
+from phl_pwd_parcels as p
+cross join lateral(
+    select 
+        s.stop_name,
+        ST_Distance(
+            ST_Transform(p.geometry, 32129),
+            ST_Transform(s.geometry, 32129)
+        )as distance_m
+    from septa_bus_stops as s
+    order by distance_m desc
+    limit 1
+) as cl;
   )
   ```
 
-4. Using the _shapes.txt_ file from GTFS bus feed, find the **two** routes with the longest trips. In the final query, give the `trip_headsign` that corresponds to the `shape_id` of this route and the length of the trip.
-
+@@ -52,26 +156,107 @@
   **Structure:**
   ```sql
   (
       trip_headsign text,  -- Headsign of the trip
       trip_length double precision  -- Length of the trip in meters
+      --Min in Sequence
+WITH min_t as (
+select shape_id, min(CAST(shape_pt_sequence as INT)) min_station
+from septa_bus_shapes
+GROUP BY shape_id
+),
+--Max in Sequence
+max_t as (
+select shape_id, max(CAST(shape_pt_sequence as INT)) max_station
+from septa_bus_shapes
+GROUP BY shape_id
+),
+-- Lat/Long
+septa_origin as (
+SELECT shape_id, shape_pt_lon, shape_pt_lat, CAST(shape_pt_sequence as INT) sequence_int
+FROM septa_bus_shapes
+),
+--Makeline, First in Sequence
+first as(
+SELECT ST_Makeline(ST_setsrid(ST_MakePoint(o.shape_pt_lon, o.shape_pt_lat), 4326) ORDER BY sequence_int)::geometry geom, o.shape_id
+FROM septa_origin o
+GROUP BY o.shape_id),
+--Makeline, Last in Sequence
+last as(
+SELECT ST_Makeline(ST_setsrid(ST_MakePoint(o.shape_pt_lon, o.shape_pt_lat),4326) ORDER BY sequence_int)::geometry geom, o.shape_id
+FROM septa_origin o
+GROUP BY o.shape_id)
+--Return 2 Longest Trips
+SELECT *, (ST_Length(geom)) FROM first
+ORDER BY st_length desc
+LIMIT 2
   )
   ```
 
 5. Rate neighborhoods by their bus stop accessibility for wheelchairs. Use Azavea's neighborhood dataset from OpenDataPhilly along with an appropriate dataset from the Septa GTFS bus feed. Use the [GTFS documentation](https://gtfs.org/reference/static/) for help. Use some creativity in the metric you devise in rating neighborhoods. Describe your accessibility metric:
 
   **Description:**
+  I don't believe this was a successful attempt. My goal here was to count the # of stops where wheelchair_access = 1 (meaning that stop is wheelchair accessible) in each neighborhood. The neighborhoods were ranking based on the number of wheelchair accessible stops therewithin.
 
+```sql
+  (
+     with stopz as(SELECT stop_id, wheelchair_boarding, ST_SetSRID(ST_MAKEPOINT(stop_lon, stop_lat), 4326) geom
+FROM septa_bus_stops),
+wc_access as(SELECT s.stop_id , s.wheelchair_boarding, s.geom, n.mapname, n.geom
+FROM stopz AS s
+JOIN neighborhoods_philadelphia as n
+ON ST_Contains(
+	n.geom,
+	s.geom))
+	
+SELECT mapname, count(stop_id)
+FROM wc_access
+WHERE 'wheelchair_boarding' = '1'
```

Comment:
The query _almost_ works. The _only_ problem is that `'wheelchair_boarding'` shouldn't be in quotes. Single quotes are only for literal values. So, `'wheelchair_boarding'` is a string with the literal value "wheelchair_boarding". Instead, if you just remove the single quotes, or use double quotes instead, you're referring to the actual values in the field.